avail/src/test/kotlin/avail/test/AvailRuntimeTestHelper.kt

```diff
@@ -342,7 +342,7 @@ class AvailRuntimeTestHelper constructor (
 		val userDir = System.getProperty("user.dir")
 		val path = userDir
 			.replace("/avail-server", "")
-			.replace("/avail-core", "")
+			.replace("/avail", "")
```

Comment:
Hm.  I think `.replace("/avail/", "/")` will work instead, to avoid over-matching (now that we don't have "avail-core" as a handy unique prefix any more).